52.md

```diff
@@ -137,15 +137,23 @@ A calendar is a collection of calendar events, represented as a custom replaceab
 
 ### Format
 
+The `.content` of these events should be a detailed description of the calendar. It is required but can be an empty string.
+
 The format uses a custom replaceable list of kind `31924` with a list of tags as described below:
-* `d` (required) calendar name
```

Comment:
This was a huge miss by me in my opinion. Calendar names should be decoupled from their identifiers. If someone wants to rename their calendar, they should be able to do so without changing the entire addressable identifier.